docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,34 @@
 # Parameters
 
-!!! warning
+The ROS packages in Autoware have ROS parameters. You need to customize the parameters depending on your applications.
+It is recommended not to set default values when declaring ROS parameters to avoid unintended behaviors due to accidental use of default values.
+Instead, set parameters from configuration files named `*.param.yaml`.
 
```

Comment:
```suggestion
Instead, set parameters from configuration files named `*.param.yaml`.

Also check out the official documentation for [understanding ROS 2 parameters](https://docs.ros.org/en/rolling/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html).

```

Maybe a small line to link to the official docs.

Reply:
Considering https://developers.google.com/style/sentence-structure, how about this?

```suggestion
Instead, set parameters from configuration files named `*.param.yaml`.

For understanding ROS 2 parameters, also check out the official documentation [Understanding parameters](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html).
```

Reply:
@xmfcx @kenji-miyake 
Thank you! Reflected: https://github.com/autowarefoundation/autoware-documentation/pull/271/commits/70f9b6c8d723c92a5f34db9bf20550db72775a21